test/integration/elsa/consumer/worker_test.exs

```diff
@@ -66,16 +70,26 @@ defmodule Elsa.Consumer.WorkerTest do
   end
 
   test "can be configured to consume the latest messages only" do
+    topic = "latest-only-topic"
+
     Patiently.wait_for!(
       fn ->
-        :ok = Elsa.create_topic(@endpoints, "latest-only-topic", partitions: 1)
+        :ok = Elsa.create_topic(@endpoints, topic, partitions: 1)
       end,
       dwell: 1_000,
       max_tries: 30
     )
 
-    Elsa.produce(@endpoints, "latest-only-topic", {"0", "strike 1"}, partition: 0)
-    Elsa.produce(@endpoints, "latest-only-topic", {"1", "strike 2"}, partition: 0)
+    Elsa.produce(@endpoints, topic, {"0", "strike 1"}, partition: 0)
+    Elsa.produce(@endpoints, topic, {"1", "strike 2"}, partition: 0)
+
+    # For this test to not fail randomly, we need to make sure the published offsets
+    # have registered with the topic before we create the consumer pointed to :latest.
+    Patiently.wait_for!(fn ->
+      {:ok, offset} = :brod.resolve_offset(@endpoints, topic, 0)
+      Logger.info("Current latest offset: #{inspect(offset)}")
+      offset == 2
+    end)
```

Comment:
Just looking at it, I believe this test was simply getting lucky before.  Elsa.produce is synchronous, but as far as I know that's no guarantee the topic will immediately report that as the latest offset when you try to resolve it.

So some slight change in timing in the new version of :brod probably made this not work all of a sudden.